include/aspect/material_model/utilities.h

```diff
@@ -28,17 +28,26 @@
 #include <deal.II/base/signaling_nan.h>
 #include <deal.II/base/parameter_handler.h>
 
+
 namespace aspect
 {
-  template <int dim> class SimulatorAccess;
```

Comment:
We typically leave this on one line, except if it becomes too long. This is just a convention of course, but please leave these lines on one line for now.